rcl/include/rcl/graph.h

```diff
@@ -158,6 +158,7 @@ rcl_names_and_types_fini(rcl_names_and_types_t * names_and_types);
  * The node_names parameter must be allocated and zero initialized.
  * The node_names is the output for this function, and contains
  * allocated memory.
+ * Note that entries in the array might contain `NULL` value.
```

Comment:
Just curious, what causes an entry to have a `NULL` value?

Reply:
A participant without a "ROS node name" will result in a `NULL` value. That can e.g. happen when the DDS implementation has some entities running all the time or is there is a vanilla DDS participant which doesn't have a node name defined in ROS.